coding-principles.md

```diff
@@ -85,4 +85,10 @@ Before placing a new system or end point live, the following points should be co
  1. Add new endpoint/system to PCI and penetration testing hosts list.
  1. Check results of next external security audit for potential issues in new system.
 
-The above would be performed in conjuction with IT and project owners would need to be mindful of the extras time this will incur before go live.
\ No newline at end of file
+The above would be performed in conjuction with IT and project owners would need to be mindful of the extras time this will incur before go live.
+
+## Linting:
+
+All new projects must have code linting enabled and configured to fail CI if the standard are not meet.
+
+The preferred way of implementing this is to see our [linting rules documentation](https://github.com/holidayextras/culture/javascript-linting-rules.md#adding-to-your-project), as this simplifies the approach provides further consistency among project.
```

Comment:
You can use a relative URL here: `[linting rules documentation](javascript-linting-rules.md#adding-to-your-project)`.